soh/CMakeLists.txt

```diff
@@ -500,6 +500,7 @@ if(MSVC)
                 /INCREMENTAL:NO;
                 /FORCE:MULTIPLE
             >
+            /MANIFEST:NO;
```

Comment:
Why `/MANIFEST:NO`? Wasn't one of this PR's goals to add the manifest? Or does this CMake option mean something else than what I'm thinking it does?

Reply:
That disables the automatic generation of the manifest.

Reply:
I'll approve this anyway, that last question was moreso asked out of curiosity.

Reply:
The manifest generated by default does not have visual styles enabled, it has this content:
```
<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v3">
    <security>
      <requestedPrivileges>
        <requestedExecutionLevel level="asInvoker" uiAccess="false"></requestedExecutionLevel>
      </requestedPrivileges>
    </security>
  </trustInfo>
</assembly>
```

More information on how visual styles are applied when using the manifest: https://learn.microsoft.com/en-us/windows/win32/controls/cookbook-overview#using-manifests-or-directives-to-ensure-that-visual-styles-can-be-applied-to-applications

Reply:
Thanks for clarifying, should be able to get this merged shortly.